CIP-0013/CIP-0013.md

```diff
@@ -1,27 +1,37 @@
 ---
 CIP: 13
 Title: Cardano URI Scheme
-Authors: Sebastien Guillemot <sebastien@emurgo.io>, Vicente Almonacid <vicente@emurgo.io>
+Authors: Sebastien Guillemot <sebastien@emurgo.io>, Vicente Almonacid <vicente@emurgo.io>, Robert Phair <rphair@cosd.com>
 Comments-URI:
 - https://github.com/Emurgo/EmIPs/pull/2
 - https://forum.cardano.org/t/cip-cardano-payment-uri-scheme/41457
+- https://github.com/cardano-foundation/CIPs/pull/25
+- https://github.com/cardano-foundation/CIPs/pull/61
+- https://forum.cardano.org/t/cip-stake-uri-scheme-for-pools-delegation-portfolios/40594
 Status: Draft
 Type: Informational
-Created: 2020-10-20
+Created: 2020-09-22
 License: CC-BY-4.0
 ---
 
 # Abstract
 
-This proposal describes a basic URI scheme to handle ADA transfers, as well
-as possible approaches for a multiplatform implementation.
+This proposal describes a basic URI scheme to handle ADA transfers and links to stake pools, as well as possible approaches for a multiplatform implementation.
 
 # Motivation
 
+#### For payment URIs: 
+
 Users who create community content often want donations as a financial incentive. However, forcing users to open their wallet and copy-paste an address lowers the amount of people likely to send tokens (especially if they have to sync their wallet first).
 
 If donating was as simple as clicking a link that opens a light wallet with pre-populated fields, users may be more willing to send tokens. URI schemes would enable users to easily make payments by simply clicking links on webpages or scanning QR Codes.
 
+#### For stake pool URIs: 
+
+Centralised sources of information, such as the Daedalus ranking algorithm and its presentation to the delegating user, have led a growing amount of stake to be disproportionately assigned to pools pushed near & beyond the saturation point.
+
+Stake pool URIs will provide a convenient and popular alternative to the trend of stake centralisation, while supporting diversity and resilience in the Cardano network.
```

Comment:
This is just an opinion. There is no way for us to know if this will be a popular alternative or not and whether or not it will actually help solve any problem. That's just what we hope will happen.

Reply:
I started addressing this in response to Vicente's review suggestion.  There are many industry examples about adoption of social features which could characterise use of stake pool links as "inevitable" if not "immediate" let alone any estimate of popularity.

So keeping the term "possible" and not wanting to remove the entire rationale for this extension, I've rephrased it without any assumptions about popular demand or cause & effect.  I do believe it can be mathematically demonstrated that offering stake pool links to 1000 marginal stake pools will support diversity even at a low level of adoption:

> Stake pool URIs will provide an additional means for small pools to acquire delegation and maintain stability, supporting diversity and possibly fault-tolerance in the Cardano network through a more even distribution of stake.